.pre-commit-hooks.yaml

```
@@ -22,3 +22,12 @@
   pass_filenames: false
   files: ^(.*/)?poetry\.lock$
   args: ["-f", "requirements.txt", "-o", "requirements.txt"]
+
+- id: poetry-install
+  name: poetry-install
+  description: Synchronize the Poetry dependencies with the lock file if out of sync
```

Another spot that reads more like `poetry install --sync`. Considering the descriptions of the other hooks, maybe:

```suggestion
  description: run poetry install to install dependencies from the lock file
```